tests/run-make/dep-graph/rmake.rs

```diff
@@ -0,0 +1,18 @@
+// Just verify that we successfully run and produce dep graphs when requested.
+
+//@ ignore-cross-compile
+
+use run_make_support::{cwd, rustc};
+
+fn main() {
+    rustc()
+        .input("foo.rs")
+        .incremental(cwd().join("incr"))
+        .arg("-Zquery-dep-graph")
+        .arg("-Zdump-dep-graph")
+        .env("RUST_DEP_GRAPH", cwd().join("dep-graph"))
+        .run();
+
+    assert!(cwd().join("dep-graph.txt").is_file());
```

Comment:
You can use the `path` helper that you have introduced recently :laughing:

Reply:
My memory doesn't go this far. :rofl: